build.gradle

```diff
@@ -269,7 +269,7 @@ subprojects { subproject ->
         testCompile "io.rest-assured:rest-assured:${restAssuredVersion}"
         testCompile group: 'org.powermock', name: 'powermock-api-mockito2', version: powermockVersion
         testCompile group: 'org.powermock', name: 'powermock-module-junit4', version: powermockVersion
-        testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.9.1'
+        testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.10.1-prerelease'
```

Comment:
This needs to be updated to a full release version of the hmcts/ccd-test-defintions before going into master. see https://github.com/hmcts/ccd-test-definitions/pull/35.